ws/src/test/scripts/org/kaazing/netx/ws/streaming/client.send.binary.payload.length.125.fragmented/handshake.response.and.frames.rpt

```diff
@@ -0,0 +1,59 @@
+accept http://localhost:8080/echo
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+read header "Host" "localhost:8080"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
+read header "Sec-WebSocket-Version" "13"
+
+write status "101" "Switching Protocols"
+write version "HTTP/1.1"
+write header "Upgrade" "websocket"
+write header "Connection" "Upgrade"
+write header "Sec-WebSocket-Accept" ${ws:handshakeHash(key)}
+
+read [0x02 0x99] ([0..4] :readMaskA)
+read option mask ${readMaskA}
+read ([0..25] :server25a)
+read option mask [0x00 0x00 0x00 0x00]
+
+read [0x00 0x99] ([0..4] :readMaskB)
+read option mask ${readMaskB}
+read ([0..25] :server25b)
+read option mask [0x00 0x00 0x00 0x00]
+
+read [0x00 0x99] ([0..4] :readMaskC)
+read option mask ${readMaskC}
+read ([0..25] :server25c)
+read option mask [0x00 0x00 0x00 0x00]
+
+read [0x00 0x99] ([0..4] :readMaskD)
+read option mask ${readMaskD}
+read ([0..25] :server25d)
+read option mask [0x00 0x00 0x00 0x00]
+
+read [0x00 0x99] ([0..4] :readMaskE)
+read option mask ${readMaskE}
+read ([0..25] :server25e)
+read option mask [0x00 0x00 0x00 0x00]
+
+read [0x80 0x80] [0..4]
+
+write [0x02 0x19]
+write ${server25a}
+
+write [0x00 0x19]
+write ${server25b}
+
+write [0x00 0x19]
+write ${server25c}
+
+write [0x00 0x19]
+write ${server25d}
+
+write [0x80 0x19]
+write ${server25e}
```

Comment:
We need to be able to test spec compliance without resorting to local K3PO scripts to handle a special case.

Instead, we need to determine if the code can approach certification by the spec scripts through a different implementation strategy, or if the spec scripts themselves need to be improved.  But, project-local scripts introduce an opportunity for divergence in implementation since a different project would not benefit from their definition.

In this case, we can probably set the max frame payload size to 25, and then have the implementation auto-flush when over that max frame size to make space in the write buffer.

Calling `close()` with 25 bytes filled up in the buffer would then flush those bytes with the `FIN` bit set, matching the existing spec tests in this area.

We may also need some additional spec scripts to handle verification of flushed non-final partial frames.  These extra spec scripts should be added to the spec project directly to be consumed by tests in the project and for the benefit of other implementations too.